Makefile

```diff
@@ -590,7 +590,7 @@ test-e2e: test-e2e-setup install-ginkgo ## Run E2E tests against OADP operator i
 	--ginkgo.vv \
 	--ginkgo.no-color=$(OPENSHIFT_CI) \
 	--ginkgo.label-filter="$(TEST_FILTER)" \
-	--ginkgo.junit-report="junit_report.xml" \
+	--ginkgo.junit-report="$(ARTIFACT_DIR)/junit_report.xml" \
```

Comment:
worked for kubevirt tests :smile:  https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_oadp-operator/1658/pull-ci-openshift-oadp-operator-master-4.19-e2e-test-kubevirt-aws/1897666246942396416/artifacts/e2e-test-kubevirt-aws/e2e/artifacts/junit_report.xml